og.module

```diff
@@ -44,6 +45,62 @@ function og_entity_insert(EntityInterface $entity) {
 }
 
 /**
+ * Implements hook_og_group_created().
+ */
+function og_og_group_created($entity_type, $bundle) {
```

Comment:
Can you add some documentation on why this hook is implemented and what is being done here? That will be helpful for people that are not familiar with this PR.